crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -239,9 +241,15 @@ impl EcmascriptModuleAssetBuilder {
     }
 
     pub async fn build_part(self, part: Vc<ModulePart>) -> Result<Vc<EcmascriptModulePartAsset>> {
+        let special_exports = self.options.await?.special_exports;
         let import_externals = self.options.await?.import_externals;
         let base = self.build();
-        Ok(EcmascriptModulePartAsset::new(base, part, import_externals))
+        Ok(EcmascriptModulePartAsset::new(
+            base,
+            part,
+            special_exports,
```

Comment:
We don't need to pass `special_exports` here. It's already available as `base.options.special_exports`